### PR TITLE
Update svc.yaml

### PR DESCRIPTION
**Pull request by jbussdieker**
Add load balancer source ranges to service

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

File: stable/kubernetes-dashboard/templates/svc.yaml [hunk 1]
```diff
@@ -28,6 +28,10 @@ spec:
 {{- end }}
 {{- if hasKey .Values.service "nodePort" }}
     nodePort: {{ .Values.service.nodePort }}
+{{- end }}
+{{- if .Values.service.LoadBalancerSourceRanges }}
+  loadBalancerSourceRanges:
+{{ toYaml .Values.service.LoadBalancerSourceRanges | indent 4 }}
```
Comment (davidkarlsen) [inline stable/kubernetes-dashboard/templates/svc.yaml]:
```suggestion
{{ toYaml .Values.service.loadBalancerSourceRanges | indent 4 }}
```

File: stable/kubernetes-dashboard/templates/svc.yaml [hunk 2]
```diff
@@ -28,6 +28,10 @@ spec:
 {{- end }}
 {{- if hasKey .Values.service "nodePort" }}
     nodePort: {{ .Values.service.nodePort }}
+{{- end }}
+{{- if .Values.service.LoadBalancerSourceRanges }}
+  loadBalancerSourceRanges:
```
Comment (davidkarlsen) [inline stable/kubernetes-dashboard/templates/svc.yaml:33]: Also add docs in README

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/kubernetes-dashboard/templates/svc.yaml (hunk 1 above)
Comment (jbussdieker) [reply stable/kubernetes-dashboard/templates/svc.yaml]: Updated

File: stable/kubernetes-dashboard/templates/svc.yaml (hunk 2 above)
Comment (jbussdieker) [reply stable/kubernetes-dashboard/templates/svc.yaml:33]: Added

**Comment by jbussdieker** [comment]
Were there any other changes needed?

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**